core/modules/layout/includes/block.text.inc

```diff
@@ -0,0 +1,69 @@
+<?php
+
+/**
+ *  TextBlock extends Block
+ *  
+ *  This class allows us to create text blocks.
+ */
+class TextBlock extends Block {
+  /**
+   *  Sets title text on draggable block panel in Layout builder.
+   */
+  function getAdminTitle() {
+    return t('Text block');
+  }
+
+  /**
+   *  Sets description text on draggable block in Layout builder.
+   */
+  function getAdminPreview() {
+    $label_text = !empty($this->settings['subject']) ? ': ' . check_plain($this->settings['subject']) : '';
+    $preview = '<p>' . t('Text') . $label_text . '</p>';
+    $preview .= $this->getAdminConditionsPreview();
```

Comment:
The description here can be "A custom text block." Or you could even remove the entire thing I think and just let the `Block::getAdminPreview()` handle it.